src/components/tables/PrefixAlerts/Rows.tsx

```diff
@@ -0,0 +1,48 @@
+import { TableCell, TableRow, useTheme } from '@mui/material';
+import AlertEditButton from 'components/admin/Settings/PrefixAlerts/EditButton';
+import ChipListCell from 'components/tables/cells/ChipList';
+import { getEntityTableRowSx } from 'context/Theme';
+import { PrefixSubscription } from 'utils/notification-utils';
+
+interface RowsProps {
+    data: [string, PrefixSubscription][];
+}
+
+interface RowProps {
+    row: [string, PrefixSubscription];
+}
+
+function Row({ row }: RowProps) {
+    const theme = useTheme();
+
+    const prefix = row[0];
+    const data = row[1];
+
+    return (
+        <TableRow hover sx={getEntityTableRowSx(theme)}>
+            <TableCell>{prefix}</TableCell>
+
+            <ChipListCell
+                values={data.userSubscriptions.map(({ email }) => email)}
+                stripPath={false}
+                maxChips={3}
+            />
+
+            <TableCell>
+                <AlertEditButton prefix={prefix} />
+            </TableCell>
```

Comment:
nit pick / thinking

The component `AlertEditButton` is only ever consumer here... this would probably make sense to make a `cell` for it to follow the pattern.